src/vendors/simple-data-table/src/index.js

```diff
@@ -4,6 +4,7 @@ class SimpleDataTable {
         this.addButtonLabel = options.addButtonLabel || '✚';
         this.defaultColumnPrefix = options.defaultColumnPrefix || 'column';
         this.defaultColumnNumber = options.defaultColumnNumber || 3;
+        this.defaultHighlightedCellClass = options.defaultHighlightedCellClass || 'highlighted-cell';
```

Comment:
Vendors canont be modified.

Reply:
Vendors are not modified. I updated 'simple-data-table' to the latest version.